src/overlays/actors/ovl_Bg_Spout_Fire/z_bg_spout_fire.h

```diff
@@ -10,7 +10,9 @@ typedef void (*BgSpoutFireActionFunc)(struct BgSpoutFire*, GlobalContext*);
 typedef struct BgSpoutFire {
     /* 0x0000 */ Actor actor;
     /* 0x0144 */ BgSpoutFireActionFunc actionFunc;
```

Comment:
```suggestion
    /* 0x000 */ Actor actor;
    /* 0x144 */ BgSpoutFireActionFunc actionFunc;
```
Seem to have forgot to remove the leading 0's again.